assets/javascript/app-no.js

```diff
@@ -0,0 +1,93 @@
+$("#start").on("click", start);
+
+document.getElementById("start").innerHTML = 003 + ":" + 20;
+startTimer();
+
+function startTimer() {
+  var presentTime = document.getElementById("start").innerHTML;
+  var timeArray = presentTime.split(/[:]+/);
+  var m = timeArray[0];
+  var s = checkSecond(timeArray[1] - 1);
+  if (s == 59) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use ===/!== to compare with true/false or Numbers](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555492)